theme/n21/views/firstLogin/sections/welcome_existing.hbs

```diff
@@ -0,0 +1,4 @@
+<section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page" {{/if}}>
```

Comment:
This file seems to be identical with the general [welcome_existing section](https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/views/firstLogin/sections/welcome_existing.hbs)? What is it needed for?

Reply:
I removed the file, after I've tested that everything works without it.